Vizzuality/i2i
# fixing jump when user clicks on show more

Fixing jump to the top of the page when the user clicks on show more button on the insights page.

## Testing instructions

Browse to the insights page, you should be redirected to `insights/all`. Then click on "Show more" button. The page shouldn't go to the top of the page and you should see more content loaded.

## Pivotal Tracker

[#169227259](https://www.pivotaltracker.com/story/show/169227259)
